packages/metascraper/test/unit/xss/xss.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+const fs = require('fs')
+const path = require('path')
+const metascraper = require(path.join(__dirname, '..', '..', '..'))([
+  require(path.join(__dirname, '..', '..', '..', '..', 'metascraper-title'))()
```

Comment:
just require('metascraper-title') is enough since when the test run the `NODE_PATH` will be the current directory. Same for `metascraper` path.